CollAction/Frontend/app/global/ShareButtons.tsx

```diff
@@ -79,31 +79,42 @@ class ShareButtons extends React.Component<IShareButtonsProps, IShareButtonsStat
     return `https://www.linkedin.com/shareArticle?mini=true&url=${this.getUrl()}&title=${this.getTitle()}&source=${encodeURIComponent(window.location.origin)}`;
   }
 
+  getWhatsappUrl() {
+      return `whatsapp://send?text=${this.getTitle()} ${this.getUrl()}`;
```

Comment:
You have to escape spaces with %20 here (the space between ${this.getTitle()} and ${this.getUrl()}). Also, might be worth a try to see if it's possible to actually send something like <a href="${this.getUrl()}">${this.getTitle()}</a>.